doc/source/quickstart/3)_Simple_Visualization.ipynb

```diff
@@ -64,7 +64,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `show` command simply sends the plot to the IPython notebook.  You can also call `p.save()` which will save the plot to the file system.  This function accepts an argument, which will be pre-prended to the filename and can be used to name it based on the width or to supply a location.\n",
+    "The `show` command simply sends the plot to the IPython notebook.  You can also call `p.save()` which will save the plot to the file system.  This function accepts an argument, which will be pre-prepended to the filename and can be used to name it based on the width or to supply a location.\n",
```

Comment:
I think this should just be "prepended".

Reply:
oops, yes, sorry. the spell checker didn't like `prended`, I knew the word was supposed to be prepended, but forgot to do that when I let my browser suggest the replacement.